packages/core/src/Stores/client-store.js

```diff
@@ -721,6 +722,10 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.includes('financial_assessment_notification');
     }
 
+    get is_poa_older_than_six_months() {
+        return this.account_status?.status?.includes('poa_older_than_six_months');
+    }
+
```

Comment:
cant we get this information from hooks directly.
Do we need this computed value then?